src/main/resources/openapi/transcriptions.yaml

```diff
@@ -951,6 +951,57 @@ paths:
               schema:
                 allOf:
                   - $ref: './problem.yaml'
+
+  /admin/transcription-documents/{transcription_document_id}/approve-deletion:
+    post:
+      tags:
+        - Transcription
+      summary: Approve deletion of a hidden transcription document file. Also known as "soft delete".
+      operationId: approveDeletionOfTranscriptionDocumentId
+      parameters:
+        - name: transcription_document_id
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
```

Comment:
Do we also want to indicate all of the 422 errors for the things like:
TRANSCRIPTION_DOCUMENT_DELETE_NOT_SUPPORTED, TRANSCRIPTION_DOCUMENT_DELETION_ALREADY_APPROVED, TRANSCRIPTION_DOCUMENT_DELETION_CAN_NOT_APPROVE_OWN_REQUEST, TOO_MANY_RESULTS, TRANSCRIPTION_DOCUMENT_MARKED_FOR_DELETION_REASON_NOT_FOUND

Reply:
done